### PR TITLE
Adds the sprint button (d) to player

### PR DESCRIPTION
**Pull request by nickhecht**
I do it for the vegans

### CODE REVIEW

File: src/client/game/Entity.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ var Entity = function(game, health, name, walkSpeed, attackSpeed) {
   this._sprite = null;
   this._direction = 'right';
   this._name = name;
-  this._WALK_SPEED = walkSpeed;
-  this._ATTACK_SPEED = attackSpeed;
+  this._WALK_FPS = walkSpeed;
+  this._ATTACK_FPS = attackSpeed;
```
Comment (jakewilson) [inline src/client/game/Entity.js:8]: This is why `attack()` is broken. See line 33.

**Comment by nickhecht** [comment]
Ya I had already fixed that before I message you guys.... Still doesn't work

**Comment by nickhecht** [comment]
Assuming you are talking about the ATTACK_SPEED -> ATTACK_FPS

**Comment by jakewilson** [comment]
On line 33, yes. Well, what do you mean by 'doesn't work'?

File: src/client/game/Player.js [hunk 2]
```diff
@@ -67,20 +67,19 @@ Player.prototype.jump = function() {
...
   if (this._currentPlayingAnim === null || this._currentPlayingAnim.name.indexOf('walk') >= 0) {
-    this._currentPlayingAnim = this._sprite.animations.play('walk' + this._direction, this._WALK_SPEED);
+    this._currentPlayingAnim = this._sprite.animations.play('walk' + this._direction, this._WALK_FPS);
   }
 }
 
 /**
- * Sets the attack animation and shoots a projectile
- */
+ * Sets the attack animation and shoots a projeD/
```
Comment (jakewilson) [inline src/client/game/Player.js]: you didn't close this comment, so the `attack()` method has been commented out. That's why the attack isn't working. Why did you change this line, anyway?

File: src/client/game/Player.js [hunk 3]
```diff
@@ -132,3 +131,10 @@ Player.prototype.update = function() {
 Player.prototype.getSprite = function() {
   return this._sprite;
 }
+
+Player.prototype._sprint = function() {
+	if(this._WALK_SPEED == 250) {
```
Comment (jakewilson) [inline src/client/game/Player.js]: 1. You should always use `===` unless you specifically are trying to use type conversion.
2. Instead of `if (this._WALK_SPEED === 250)` let's do `if (this._sprinting)`.
3. Please put a space in between the `if` statement and `(`.

File: src/client/game/Player.js [hunk 4]
```diff
@@ -132,3 +131,10 @@ Player.prototype.update = function() {
...
+		this._WALK_SPEED = this._WALK_SPEED * 5;
+	} else
+		this._WALK_SPEED = this._WALK_SPEED / 5;
```
Comment (jakewilson) [inline src/client/game/Player.js:147]: Please add braces to this `else`.

File: src/client/game/Player.js [hunk 5]
```diff
@@ -80,6 +81,7 @@ Player.prototype.move = function(direction) {
 
 /**
  * Sets the attack animation and shoots a projeD/
```
Comment (jakewilson) [inline src/client/game/Player.js]: Please put this line back to how it was before. What's a "projeD"?

File: src/client/game/Player.js [hunk 6]
```diff
@@ -132,3 +133,13 @@ Player.prototype.update = function() {
...
+
+Player.prototype._sprint = function() {
+	if (this._sprinting) {
+		this._WALK_SPEED = this._WALK_SPEED * 5;
+ 		this._sprinting = true; 
+	} else {
+		this._WALK_SPEED = this._WALK_SPEED / 5;
+ 		this._sprinting = false; 
+	}
+}
```
Comment (jakewilson) [inline src/client/game/Player.js:150]: You didn't test this.

**Comment by jakewilson** [comment]
:+1:
